PKPD/gui/mainWindow.py

```diff
@@ -174,6 +175,16 @@ def next_tab(self):
                 # generate error message
                 error_message = 'The .csv file does not seem to be properly formatted. Please check again!'
                 QtWidgets.QMessageBox.question(self, 'Data structure not compatible!', error_message, QtWidgets.QMessageBox.Yes)
+
+            #Check Units in MMT file
```

Comment:
to be consistent through out the program: space between # and comment, i.e.

`# check units in mmt file`